go/vt/sqlparser/ast.go

```diff
@@ -499,22 +499,25 @@ func (node *Set) WalkSubtree(visit Visit) error {
 }
 
 // DDL represents a CREATE, ALTER, DROP or RENAME statement.
-// Table is set for AlterStr, DropStr, RenameStr.
+// Table is set for AlterStr, DropStr, RenameStr, ReorganizeStr
 // NewName is set for AlterStr, CreateStr, RenameStr.
 type DDL struct {
-	Action    string
-	Table     TableName
-	NewName   TableName
-	IfExists  bool
-	TableSpec *TableSpec
+	Action               string
+	Table                TableName
+	NewName              TableName
+	IfExists             bool
+	TableSpec            *TableSpec
+	PartitionName        ColIdent
```

Comment:
let's keep it like this for now. i think the more future-proof thing to do, which i don't want to spend time on right now, is to:
* make DDL an interface
* make Alter a type (along with Create, Drop, Rename, etc)
* on the Alter type, have a PartitionSpec that can be reused betweent the Create+Alter types